bin/configs/go-petstore-withXml.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: go
+outputDir: samples/openapi3/client/petstore/go-petstore-withXml
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore-addpet-only.yaml
+templateDir: modules/openapi-generator/src/main/resources/go
+additionalProperties:
+  enumClassPrefix: "true"
+  packageName: petstore
+  disallowAdditionalPropertiesIfNotPresent: false
+  withXml: false
```

Comment:
that looks wrong, should't that be `withXml: true`? When I look at the generated code the xml attributes are missing too

Reply:
good catch. forgot to commit the updated code with withXml set to true

will do it shortly